src/helpers/helper-functions.js

```diff
@@ -21,4 +21,18 @@ module.exports = {
         return item;
       });
   },
+  /**
+  *
+  * @param comments
+  * @returns []
+  */
+  getAnnotations(comments) {
+    return comments.map((c) => ({
+      path: c.path,
+      start_line: c.line_no ? c.line_no : 1,
+      end_line: c.line_no ? c.line_no : 1,
+      annotation_level: 'warning',
+      message: 'Problematic comment identified',
```

Comment:
Maybe just in the spirit of accuracy / giving the dev a hint as to how to fix the problem:

```suggestion
      message: 'Problematic comment ("TODO", "HACK", "FIXME") identified',
```

Reply:
Done! Added The necessary changes